net-misc/sipp/sipp-3.6.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SIPp is a free Open Source test tool / traffic generator for the SIP protocol"
+HOMEPAGE="http://sipp.sourceforge.net/ https://github.com/SIPp/sipp/releases"
+SRC_URI="https://github.com/SIPp/sipp/releases/download/v${PV}/${PF}.tar.gz"
+
+LICENSE="GPL-2 ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl +pcap sctp +ssl"
+
+DEPEND="sys-libs/ncurses:=
+	ssl? ( dev-libs/openssl:= )
+	pcap? ( net-libs/libpcap net-libs/libnet:1.1 )
```

Comment:
Please multi-line the deps, I didn't even see libnet requirement.

Reply:
Done.